aws-frauddetector-detector/src/aws_frauddetector_detector/tests/helpers/test_model_helpers.py

```diff
@@ -74,11 +74,15 @@ def test_get_model_for_detector():
     mock_afd_client.get_rules = MagicMock(return_value=get_rules_response)
     mock_afd_client.get_outcomes = MagicMock(return_value=get_outcomes_response)
     mock_afd_client.get_external_models = MagicMock(return_value=get_external_models_response)
+    mock_afd_client.get_external_models = MagicMock(return_value=get_external_models_response)
```

Comment:
Don't need this duplicate line (and don't need to mock get_model_versions, since model version arn is returned in get_detector_version)

Reply:
Good catch. Removing, I was initially going after an API call but then it made sense to rely on the response from `GetDetectorVersion`